src/ipc/dma-copy.c

```diff
@@ -154,6 +154,9 @@ int dma_copy_new(struct dma_copy *dc)
 	}
 
 #if !CONFIG_DMA_GW
+	/* free previous used channel */
+	if (dc->chan)
```

Comment:
Lets free this directly using reset or a new API (called via IPC DMA trace disable). This is just a hack.

Reply:
Can't agree more, actually, this was already in my commit message :)